auth-web/tests/unit/test-utils/test-data/MockI18n.ts

```diff
@@ -0,0 +1,22 @@
+import { castToVueI18n, createI18n } from 'vue-i18n-bridge'
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
+
+export default {
+  mock () {
+    Vue.use(VueI18n, { bridge: true })
+
+    const i18n = castToVueI18n(createI18n({
+      legacy: false,
+      locale: 'en',
+      fallbackLocale: 'en',
+      messages: {
+        en: {
```

Comment:
set this to the given param (this will allow other test suites to use this with custom data)